keps/sig-api-machinery/3488-cel-admission-control/kep.yaml

```diff
@@ -43,6 +43,7 @@ latest-milestone: "v1.27"
 # The milestone at which this feature was, or is targeted to be, at each stage.
 milestone:
   alpha: "v1.26"
+  beta: "1.28"
```

Comment:
Shouldn't it be 1.27? That's the current one you're targeting right?

Reply:
No, we are doing another alpha in 1.27. Not promoting to next stage

Reply:
Sorry, that's uncommon so I missed it! Thanks!